.github/workflows/pull_request.yml

```diff
@@ -25,7 +25,14 @@ jobs:
         run: npm ci
       - name: Install Playwright Browsers
         run: npx playwright install --with-deps
+      - name: Pull Latest docker image
+        run: docker pull amazon/dynamodb-local:latest
       - name: Run Playwright tests
+        env:
+          ARC_TABLES_PORT: 8000
+          ARC_DB_EXTERNAL: true
+          AWS_ACCESS_KEY_ID: fakeTestString
+          AWS_SECRET_ACCESS_KEY: fakeTestString
```

Comment:
No longer necessary due to bug fixes in your client.

```suggestion
```

Reply:
This still crashes if these are missing

Reply:
Is it working now?